public/app/features/geo/utils/frameVectorSource.ts

```diff
@@ -0,0 +1,45 @@
+import { DataFrame } from '@grafana/data';
+import { Feature } from 'ol';
+import { Geometry } from 'ol/geom';
+import VectorSource from 'ol/source/Vector';
+import { getGeometryField, LocationFieldMatchers } from './location';
+
+export interface FrameVectorSourceOptions {}
+
+export class FrameVectorSource extends VectorSource<Geometry> {
+  constructor(private location: LocationFieldMatchers) {
+    super({
+      // attributions?: import("./Source.js").AttributionLike;
+      // features?: Array<Feature<any>> | Collection<Feature<any>>;
+      // format?: import("../format/Feature.js").default;
+      // loader?: import("../featureloader.js").FeatureLoader;
+      // overlaps?: boolean;
+      // strategy?: LoadingStrategy;
+      // url?: string | import("../featureloader.js").FeatureUrlFunction;
+      // useSpatialIndex?: boolean;
+      // wrapX?: boolean;
```

Comment:
Are these placeholders for future extended functionality? Are these comments needed?

Reply:
they are really just copy+paste from the docs, so I knew what options exist.

I'll remove now